am/README.adoc

```diff
@@ -107,6 +107,13 @@ mongo:
   uri: kubernetes://default/secrets/mongo/mongouri
 ----
 
+[TIP]
+====
+If you need to access a secret, you have to create a role within your namespace.
+
+If you need to access a secret in another namespace, you have to create a role in that namespace. The two roles can have the same name, but they are completely seperate objects.
```

Comment:
```suggestion
If you are deploying in another namespace and you need to access a secret there, you have to create a separate role in that namespace. The two roles can have the same name, but they are completely separate objects - each role only gives access to the namespace it is created in.
```

Reply:
The suggested change are fine for me. Do we have to give the needed parameter to create this role or this have to go in an other part of the documentation? 
```
apim:
  managedServiceAccount: true
  roleRules:
    - apiGroups:
        - ""
      resources:
        - configmaps
        - secrets
      verbs:
        - get
        - list
        - watch
```

Reply:
In my view if this is a helm/k8s thing we should just reference the proprietary documentation, e.g. https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example

WDYT?